generators/Exercises/Yacht.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class Yacht : GeneratorExercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
```

Comment:
At the moment, this overridden method doesn't do anything, so it can be removed.